great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -153,14 +139,19 @@ def __init__(self, *args, **kwargs) -> None:
         self._data_sampler = PandasDataSampler()
 
     def _instantiate_azure_client(self) -> None:
-        azure_options = self.config.get("azure_options", {})
-        try:
-            if "conn_str" in azure_options:
-                self._azure = BlobServiceClient.from_connection_string(**azure_options)
-            else:
-                self._azure = BlobServiceClient(**azure_options)
-        except (TypeError, AttributeError):
-            self._azure = None
+        self._azure = None
+        if BlobServiceClient:
+            azure_options = self.config.get("azure_options", {})
+            try:
+                if "conn_str" in azure_options:
+                    self._azure = BlobServiceClient.from_connection_string(
+                        **azure_options
+                    )
+                else:
+                    self._azure = BlobServiceClient(**azure_options)
+            except (TypeError, AttributeError):
```

Comment:
I think you can remove this try/except since there is the `if BlobServiceClient` check above

Reply:
@anthonyburdi I tried  and it did not work; upon experimentation, I learned that it is doing some other checking -- that `TypeError` is important.  Thanks.